Source/Core/DolphinWX/VideoConfigDiag.h

```diff
@@ -193,6 +193,8 @@ class VideoConfigDiag : public wxDialog
 		// Anti-aliasing
 		choice_aamode->Enable(vconfig.backend_info.AAModes.size() > 1);
 		text_aamode->Enable(vconfig.backend_info.AAModes.size() > 1);
+		if (vconfig.backend_info.bSupportsSSAA)
+			ssaa_checkbox->Enable(vconfig.iMultisampleMode > 0);
```

Comment:
Adding an else with ssaa_checkbox->Disable(); returns a grey checkbox in Direct3D.